src/OrchardCore.Modules/OrchardCore.Sms.Azure/Startup.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.Modules;
+using OrchardCore.Security.Permissions;
+using OrchardCore.Settings;
+using OrchardCore.Sms.Azure.Drivers;
+
+namespace OrchardCore.Sms.Azure;
+
+public class Startup : StartupBase
+{
+    private readonly IHostEnvironment _hostEnvironment;
+
+    public Startup(IHostEnvironment hostEnvironment)
+    {
+        _hostEnvironment = hostEnvironment;
+    }
+
+    public override void ConfigureServices(IServiceCollection services)
+    {
+        services.AddSmsServices();
+        services.AddPhoneFormatValidator();
+
+        if (_hostEnvironment.IsDevelopment())
+        {
+            services.AddLogSmsProvider();
+        }
```

Comment:
this isn't needed here. it already exists in the OrchardCore.Sms module

Reply:
Oops, I forgot to remove this. It's my fault